### PR TITLE
Blog post for the barcamp kick-off

### PR DESCRIPTION
**Pull request by wetneb**
Draft of blog post to announce the initial schedule of the BarCamp.

### CODE REVIEW

File: blog/2024-06-01-barcamp-registration.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+This will give you access to the video conference room, let you propose and vote on sessions.
+
+The event will start Monday afternoon:
+* 14:00 CEST (12:00 UTC): Introductions round and opening of the event.
+* 14:30 CEST (12:30 UTC): Collaborative review and scheduling of [the proposed sessions](https://barcamps.eu/openrefine-2024-barcamp/sessions).
+* 16:00 CEST (14:00 UTC): Break
+* 16:30 CEST (14:30 UTC): Allocation of volunteering shifts (session facilitation, note-taking, Zoom caretaking)
+
+The sessions themselves will then take place on Tuesday and Wednesday, following the schedule established on Monday.
+One exception is the session about the mission, visions and values of OpenRefine facilitated by BoCoup, taking place on Tuesday at XX CEST (XX UTC), already scheduled because of the external facilitation.
```
Comment (wetneb) [inline blog/2024-06-01-barcamp-registration.md:17]: @magdmartin when is the BoCoup session again?

File: blog/2024-06-01-barcamp-registration.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Our first ever BarCamp is coming up in two weeks, on June 17th-19th, in Berlin and online.
+[Register now](https://barcamps.eu/openrefine-2024-barcamp/) to participate, whether you want to be onsite or remotely.
+This will give you access to the video conference room, let you propose and vote on sessions.
+
+The event will start Monday afternoon:
+* 14:00 CEST (12:00 UTC): Introductions round and opening of the event.
+* 14:30 CEST (12:30 UTC): Collaborative review and scheduling of [the proposed sessions](https://barcamps.eu/openrefine-2024-barcamp/sessions).
+* 16:00 CEST (14:00 UTC): Break
+* 16:30 CEST (14:30 UTC): Allocation of volunteering shifts (session facilitation, note-taking, Zoom caretaking)
```
Comment (wetneb) [inline blog/2024-06-01-barcamp-registration.md:14]: Random schedule made up when writing the blog post, we can adjust as you see fit.
@ayushrai206 I think it's easier to do the ice-breaker separately (later in the evening, separately for online and in-person attendees). See https://github.com/OpenRefine/BarCamp2024/issues/13

File: blog/2024-06-01-barcamp-registration.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+author: Antonin Delpeuch
+title: "Register to participate to our BarCamp remotely"
+---
+
+Our first ever BarCamp is coming up in two weeks, on June 17th-19th, in Berlin and online.
+[Register now](https://barcamps.eu/openrefine-2024-barcamp/) to participate, whether you want to be onsite or remotely.
```
Comment (ayushrai206) [inline blog/2024-06-01-barcamp-registration.md:7]: Register now, if not already done yet.

File: blog/2024-06-01-barcamp-registration.md (hunk 2 above)
Comment (wetneb) [reply blog/2024-06-01-barcamp-registration.md:14]: Ah yes if it's indeed something rather short that can fit in an introduction round, then that makes sense (I thought you had something longer in mind)

File: blog/2024-06-01-barcamp-registration.md (hunk 1 above)
Comment (magdmartin) [reply blog/2024-06-01-barcamp-registration.md:17]: The session will be on  June 18th. We still need to agree on the exact time; we are looking for a two hours between 14:00 CEST and 18:00 CEST. The exact start time will depend on what fit best the remote attendees.

File: blog/2024-06-01-barcamp-registration.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+Our first ever BarCamp is coming up in two weeks, on June 17th-19th, in Berlin and online.
+If you haven't yet, [register now](https://barcamps.eu/openrefine-2024-barcamp/) to participate, whether you want to be onsite or remotely.
+This will give you access to the video conference room, let you propose and vote on sessions.
+
+The event will start Monday afternoon:
+* 14:00 CEST (12:00 UTC): Introductions round and opening of the event.
+* 14:30 CEST (12:30 UTC): Collaborative review and scheduling of [the proposed sessions](https://barcamps.eu/openrefine-2024-barcamp/sessions).
+* 16:00 CEST (14:00 UTC): Break
+* 16:30 CEST (14:30 UTC): Allocation of volunteering shifts (session facilitation, note-taking, Zoom caretaking)
```
Comment (magdmartin) [inline blog/2024-06-01-barcamp-registration.md:14]: Should we add a link to https://github.com/OpenRefine/BarCamp2024/issues/11 in the blogpost?

**Pull request merged**